<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget1</class>
 <widget class="QWidget" name="Widget1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget1</string>
  </property>
  <widget class="QTextBrowser" name="text3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:48pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>312</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="buton">
      <property name="text">
       <string>toplaaaaaa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Çıkar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Çarp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Böl</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
